socorro/external/rabbitmq/crashstorage.py

```diff
@@ -299,3 +303,33 @@ class ReprocessingRabbitMQCrashStore(RabbitMQCrashStorage):
         False
     )
 
+
+#==============================================================================
+class ReprocessingOneRabbitMQCrashStore(ReprocessingRabbitMQCrashStore):
+    required_config = Namespace()
+    required_config.rabbitmq_class = change_default(
+        RabbitMQCrashStorage,
+        'rabbitmq_class',
+        ConnectionContext,
+    )
+    required_config.routing_key = change_default(
+        RabbitMQCrashStorage,
+        'routing_key',
+        'socorro.reprocessing'
+    )
+
+    def __init__(self, config, quit_check_callback=None):
+        config.forbidden_keys = ''
+        config.redactor_class = Redactor
+        config.throttle = 100
+        config.filter_on_legacy_processing = False  # must be set
```

Comment:
These 4 lines I just don't understand why they're not set by default. Grrr...
You'd think that making an instance of this class would create a `config` that has the values of its parent class by default. 
If you delete any of these lines you get subtle `KeyError`s further down.